apps/web/serializers.py

```diff
@@ -16,3 +16,10 @@ class Meta:
         model = Team
         fields = ('name', 'description', 'headshot', 'background_image', 'github_url', 'linkedin_url',
                   'personal_website', 'team_type')
+
+
+class ContributorSerializer(serializers.ModelSerializer):
```

Comment:
Can you just use the `TeamSerializer`? There is no need to create a new serializer.